Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -7937,3 +7937,213 @@
+  client:
+    type: browser
+    name: Via
+    version: ""
+    engine: Blink
+    engine_version: 101.0.0001.320060
+    family: Chrome
+  headers:
+    http-x-requested-with: mark.via.gq
+-
+  user_agent: Mozilla/5.0 (Linux; Android 11; Pixel 5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.91 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Via
+    version: ""
+    engine: Blink
+    engine_version: 90.0.4430.91
+    family: Chrome
+  headers:
+    http-x-requested-with: mark.via.pm
+-
+  user_agent: Mozilla/5.0 (iPhone; CPU iOS 15.6.1 like Mac OS X) AppleWebKit/613.3.9.0.16 (KHTML, like Gecko) Mobile/15E148 Nuviu/1.0.5
+  client:
+    type: browser
+    name: Nuviu
+    version: 1.0.5
+    engine: WebKit
+    engine_version: 613.3.9.0.16
+    family:
+-
+  user_agent: fGet/2 CFNetwork/1485 Darwin/23.1.0
+  client:
+    type: browser
+    name: fGet
+    version:
+    engine:
+    engine_version:
+    family:
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10)
+  client:
+    type: browser
+    name: Thor
+    version: ""
+    engine: WebKit
+    engine_version: ""
+    family: Android Browser
+  headers:
+    http-x-requested-with: threads.thor
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.5735.130 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Incognito Browser
+    version: ""
+    engine: Blink
+    engine_version: 114.0.5735.130
+    family: Chrome
+  headers:
+    http-x-requested-with: com.androidbull.incognito.browser
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Godzilla Browser
+    version: ""
+    engine: Blink
+    engine_version: 92.0.4515.159
+    family: Chrome
+  headers:
+    http-x-requested-with: com.mosoft.godzilla
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/112.0.5615.101 Mobile DuckDuckGo/105 Safari/537.36
+  client:
+    type: browser
+    name: Ocean Browser
+    version: ""
+    engine: Blink
+    engine_version: 112.0.5615.101
+    family: Chrome
+  headers:
+    http-x-requested-with: com.oceanbrowser.mobile.android
+-
+  user_agent: Mozilla/5.0 (Linux; Android 12) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/103.0.5060.129 Mobile DuckDuckGo/1 Safari/537.36
+  client:
+    type: browser
+    name: Qmamu
+    version: ""
+    engine: Blink
+    engine_version: 103.0.5060.129
+    family: Chrome
+  headers:
+    http-x-requested-with: com.qmamu.browser
+-
+  user_agent: Mozilla/5.0 (Linux; Android 12) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/111.0.5563.58 Mobile DuckDuckGo/17 Safari/537.36
+  client:
+    type: browser
+    name: BF Browser
```

Comment:
We have a `BF Browser` registered with `com.bf.browser`, which now is called `BXE Browser`. They are not related, though, so I guess it would be better to also rename `com.bf.browser` while adding this one.
At the same time there are also two clones with `com.befaster.bfbrowser` and `com.befaster.bfbrowser`. They are definitively clones of each other, but I also think they are clones of this this new BF Browser, but slightly different versions. I mean `com.techlastudio.bfbrowser` description is largely the same as one of those clones and, and it does say "the latest version", which may mean that it's the latest clone. It may be worth it to add IDs from the older versions.